addon/package.json

```diff
@@ -36,6 +36,13 @@
   },
   "preferences": [
     {
+      "name": "WHITELIST_URLS",
```

Comment:
Well, that's why I also mentioned [sticking this in the list of canned server environments](https://github.com/ckprice/testpilot/blob/issue-463-content-detection/addon/index.js#L47). Main reason to modify this is for testing, and we expose the local/stage/prod choice to the user. So, that might be a happy medium rather than exposing the list completely.